src/main/java/msjava2lesson/Main.java

```diff
@@ -1,7 +1,370 @@
 package msjava2lesson;
 
+import msjava2lesson.moveZeroes.Solution;
+import msjava2lesson.reverselinkedlist.ListNode;
+
+import java.util.Arrays;
+
 public class Main {
+
     public static void main(String[] args) {
-        System.out.println("Hello World!");
+
+        taskReverseLinkedList();
+        taskMoveZeros();
+    }
+
+    private static void linkedListPrint(ListNode nodeStart) {
+
+        boolean isStart = true;
+        while (nodeStart != null){
+
+            if(isStart)
+                System.out.print("Head:");
+            isStart = false;
+
+            System.out.print(nodeStart.val + " -> ");
+            nodeStart = nodeStart.next;
+        }
+        System.out.println("No pointer, the end...");
+    }
+
+    /**
+     * https://leetcode.com/problems/reverse-linked-list/description/
+     */
+    private static void taskReverseLinkedList() {
+        firstWayToFillLinkedListAndReverse();
+        secondWayToFillLinkedListAndReverse();
+        thirdWayToFillLinkedListAndReverse();
+        forthWayToFillLinkedListAndReverse();
+        oneElementInLinkedListAndReverse();
+        zeroElementInLinkedListAndReverse();
+
+        // Questions
+        String[] s = new String[]{"sasd"};
+        String[] ss = {"asd"}; // is present the difference between two of them?
+    }
+
+    private static void zeroElementInLinkedListAndReverse() {
+        ListNode nodeOneElement = null;
+
+        // Show original list.
+        System.out.println("iteratively");
+        System.out.print("Origin:   ");
+        linkedListPrint(nodeOneElement);
+
+        // Reverse
+        msjava2lesson.reverselinkedlist.Solution reverseSolution = new msjava2lesson.reverselinkedlist.Solution();
```

Comment:
No need to specify the full path. If you have two classes with name `Solution` consider renaming them
```suggestion
        Solution reverseSolution = new Solution();
```

Reply:
I have class named Solution in two namespaces: 
 - reverselinkedlist
 - moveZeroes
So,I am not sure I can do like you suggested.

Reply:
I have class named Solution in two namespaces:

reverselinkedlist
moveZeroes
So,I am not sure I can do like you suggested.

Reply:
I have the same class named Solution in two different namespaces:
 - moveZeroes
 - reverselinkedlist
 So, I think it is incorrect to do like you suggested in the Main class.